### ISSUE TITLE
[feature-request] support different line cap style (especially round-end)

### ISSUE THREAD

**Issue by SteveKChiu** [nyorain/rvg#8]
First thanks for your great work.
I notice rvg does not support different line cap style, it seems it is a common feature for vector drawing library.
It would be great if the rvg::Shape can support round line cap style for non-looped path line.

**Comment by nyorain** [comment]
I agree, line caps, corner beveling and all these standard vector graphics features are on my [todo](/nyorain/rvg/blob/master/docs/roadmap.md) list, it will be implemented in the more abstract shape-baking library [katachi](/nyorain/katachi). As the feature is not very high up in the list though (and I don't got much time to work on this rn), don't expect this in the near future. Always open for PRs though :)
